frontend/src/data/config/MetricConfig.ts

```diff
@@ -2,6 +2,7 @@
 export type MetricId =
   | "population"
   | "population_pct"
+  | "my_population_pct"
```

Comment:
I wonder if a better name for this might be "brfss_population_pct" ?

Reply:
+1